### PR TITLE
UI - Document Upload Fixes, Listing Url and Single Checkbox

### PR DESCRIPTION
**Pull request by shaangill025**
*Issue:*

- https://github.com/bcgov/entity/issues/23856
- https://github.com/bcgov/entity/issues/23951
- https://github.com/bcgov/entity/issues/23685
- https://github.com/bcgov/entity/issues/24283

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-305-mnpudtqu.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-305-mnpudtqu.web.app

File: strr-web/components/bcros/form-section/review/Form.vue [hunk 1]
```diff
@@ -75,46 +75,67 @@
...
+              <div v-if="formState.isPropertyManagerRole" data-test-id="host-auth-confirmation">
+                <p>
+                  <span class="font-bold">{{ tReview('confirmHostAuthorizationLabel') }}</span>
+                  {{ tReview('confirmHostAuthorization') }}
+                </p>
+                <hr class="mt-4 border-gray-500">
+              </div>
+              <UCheckbox
+                v-model="confirmationCheckboxValue"
+                label="I confirm that I understand and agree to the above."
```
Comment (dimak1) [inline strr-web/components/bcros/form-section/review/Form.vue:130]: It should come from the translation file.

Comment (shaangill025) [reply strr-web/components/bcros/form-section/review/Form.vue]: Fixed

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-305-mnpudtqu.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-305-mnpudtqu.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-305-mnpudtqu.web.app

**Review by dimak1 — APPROVED** [review]
Looks good to me.

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-305-mnpudtqu.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-305-mnpudtqu.web.app

**Pull request merged**